### ISSUE TITLE
ampersand in file path causes failure to load shapefile on Windows

### ISSUE THREAD

**Issue by leffjesh** [mapbox/mapbox-studio-classic#1290]
Using Mapbox Studio 0.2.7 on Windows, I was unable to add a shapefile to a source project when the shapefile had an `&` in the file path.  In my case it was in a folder named `T&E Species`.  Renaming the folder to remove the `&` fixed the issue.

**Comment by springmeyer** [comment]
Thanks for the report: can you describe the exact point that failed? Was it that the `select` button was unresponsive? Basically curious if this is the same as #1277

**Comment by leffjesh** [comment]
yes, select button was unresponsive when cutting a pasting a full path to shapefile and when browsing to a folder and selecting a shapefile with an ampersand in the name.   no error reported.  When browsing folders, a folder with an ampersand doesn't allow me to double click and explore its contents.
